### PR TITLE
Adjust sync cycle

### PR DESCRIPTION
**Pull request by jetfuel**
Roughly keep the sync happens every 15~25 seconds.

### CODE REVIEW

**Comment by jetfuel** [comment]
fixes https://github.com/PaddlePaddle/VisualDL/issues/326

File: visualdl/logic/im.cc [hunk 1]
```diff
@@ -30,10 +39,23 @@ void SimpleWriteSyncGuard<T>::Start() {
 
 template <typename T>
 void SimpleWriteSyncGuard<T>::End() {
-  CHECK(data_);
-  if (data_->parent()->meta.ToSync()) {
-    Sync();
-  }
+    CHECK(data_);
```
Comment (Superjomn) [inline visualdl/logic/im.cc]: The c++ code should follow google c style, indent 2 spaces not 4 spaces.

`clang-format` can format c++ code automatically.

**Review by Superjomn — COMMENTED** [review]
Logic seems good, but need to follow google c style.

**Comment by jetfuel** [comment]
Updated.

**Review by Superjomn — APPROVED** [review]
LGTM

**Pull request merged**